src/reporter/tests/test_subscribe.py

```diff
@@ -155,5 +155,5 @@ def test_custom_time_index(clean_mongo, clean_crate):
                 TIME_INDEX_HEADER_NAME: 'my-time-index'
             }
         },
-        'metadata': ['dateCreated', 'dateModified', 'TimeInstant'],
+        'metadata': ['dateCreated', 'dateModified', 'TimeInstant', 'timestamp', 'my-time-index']
```

Comment:
are these additions to the metadata part of the subscription really needed? what's the consequence of adding them? (i.e, is Orion doing anything with those?)

Reply:
I have understood it so that only those metadata attributes that are in the subscription list are included by Orion in the notification. For example, checking for <code>timestamp</code> in metadata (as in step 6) is useless unless it is included in subscription list.

Reply:
I see, let me better test and understand what's the implications of adding these things to the metadata part. In other words, if we don't add these new items, does something change? I'll check and get back to you.

Reply:
see my comment above